internal/action/actions.go

```diff
@@ -1746,6 +1747,7 @@ func (h *BufPane) CursorPageDown() bool {
 	}
 	if selectionEndNewline {
 		h.Cursor.Start()
+		h.MoveCursorUp(1)
```

Comment:
And when it comes to `CursorPageDown`, I'm not sure this is useful. For example: put a cursor somewhere close to the end of buffer (just a few lines above the end), select some lines and then press PageDown -> the cursor is unexpectedly placed not at the last line but at the line before it.

Reply:
Indeed, that's a valid point.